setup.py

```diff
@@ -45,8 +44,8 @@
         "Operating System :: OS Independent",
         "Programming Language :: Python",
         "Intended Audience :: Science/Research",
-#        "Topic :: Formats and Protocols :: Data Formats",
-#        "Topic :: Scientific/Engineering :: Earth Sciences",
-#        "Topic :: Software Development :: Libraries :: Python Modules"
+        # "Topic :: Formats and Protocols :: Data Formats",
+        # "Topic :: Scientific/Engineering :: Earth Sciences",
+        # "Topic :: Software Development :: Libraries :: Python Modules"
         ]
```

Comment:
we could uncomment these if appropriate

Reply:
Yeah, I am unsure.  Gonna punt on this for now.